src/containers/UserProfile.js

```diff
@@ -0,0 +1,25 @@
+import { connect } from 'react-redux';
+import UserProfile from '../components/user/UserProfile';
+import {
+  fetchProfile,
+  updateProfile,
+} from '../store/actions/profileActions';
+
+const mapStateToProps = state => ({
+  profileData: state.profile.profileData,
+  profileUpdated: state.profile.profileUpdated,
+  errors: state.profile.errors,
+  error: state.profile.error,
+  authenticated: state.profile.authenticated,
+});
+
+export const mapDispatchToProps = dispatch => ({
+  fetchProfile: () => {
+    dispatch(fetchProfile());
+  },
+  updateProfile: (profileUpdateInfo) => {
+    dispatch(updateProfile(profileUpdateInfo));
+  },
+});
+
+export default connect(mapStateToProps, mapDispatchToProps)(UserProfile);
```

Comment:
This export could have been done elsewhere. Why are you making it in another file?

Reply:
The same case for this, just asking if you meant that all these exports should happen in one file.